third_party/intellij-plugins-dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcessZ.java

```diff
@@ -282,7 +282,9 @@ public XDebuggerEditorsProvider getEditorsProvider() {
   }
 
   public boolean isRemoteDebug() {
-    return myRemoteDebug;
+    // TODO(devoncarew): This is a hack for now - we want to instead look for prefix matches
+    // if remote debugging, or running a Flutter app.
```

Comment:
Could you explain this a bit more? If we will need to know about remote debugging then references to myRemoteDebug may be useful to preserve. As you know, this class will need to be reimplemented, and keeping internal notes like that around might make it easier.